test/lib/next-test-utils.js

```diff
@@ -113,6 +113,13 @@ export function renderViaHTTP(appPort, pathname, query, opts) {
   return fetchViaHTTP(appPort, pathname, query, opts).then((res) => res.text())
 }
 
+/**
+ * @param {number|string} appPort
+ * @param {string} [pathname]
+ * @param {string} [query]
+ * @param {RequestInit} opts
```

Comment:
```suggestion
 * @param {RequestInit} [opts]
```

`opts` is also an optional parameter.

Reply:
Not a TS file, just added for autocompletion, but thanks! :green_heart: